xml/install_quick.xml

```diff
@@ -7,148 +7,46 @@
   <!ENTITY % entities SYSTEM "entity-decl.ent">
     %entities;
 ]>
```

Comment:
This chapter is the initial quickstart guide. You have added the system requirements and configuration chapters before and after this chapter without adjusting the structure and content. I will not make any further inline comments to this chapter, but the following needs to be changed:

* Title
* Structure: get rid of the lone sect1 and lift all current sect2 to sect1
* formatting: please run daps-xmlormat -i on this file.Single line paras make reviewing the pull request a real pain. Configure your editor to create linebreaks at 79 characters
* In installation procedures, images should come before the text
* the image "Initial CaaS Platform Configuration" has been asymmetrically cropped

Reply:
Done.